roles/ceph-mon/tasks/deploy_monitors.yml

```
@@ -1,4 +1,15 @@
 ---
+- name: check if monitor initial keyring already exists
+  command: >
+    {{ docker_exec_cmd }} ceph --cluster ceph --name mon. -k
+    /var/lib/ceph/mon/{{ cluster }}-{{ ansible_hostname }}/keyring auth
```

```suggestion
    /var/lib/ceph/mon/{{ cluster }}-{{ hostvars[groups[mon_group_name][0]]['ansible_hostname'] }}/keyring auth
```
this will always be run on the node `groups[mon_group_name][0]` via the `delegate_to` instruction.